usl_pipeline/usl_lib/usl_lib/writers/polygon_writers.py

```diff
@@ -0,0 +1,40 @@
+import typing
+from typing import Tuple
+
+from shapely import geometry
+
+
+def num_list_to_strings(num_list):
+    return list(map(lambda v: str(v), num_list))
+
+
+def write_polygons_to_text_file(
+    polygons_with_masks: list[Tuple[geometry.Polygon, int]],
```

Comment:
I think this is an `Iterable` now rather than a `list`?

Reply:
It's not convenient for me here to deal with iterable here since I need to write the number of polygons in the first output line. Otherwise I would have to convert iterable to the list internally which I don't want to do (it's better to do it outside of the function and might not be needed at all for cases when the caller already has a list to send as an input). As I mentioned before, the splitting logic in chunker collects lists of polygons for each chunk and that's fine (not too much memory is needed for keeping all the polygons in memory at the same time). So let's not over-complicate the logic here?